content/geoip/docs/databases/city-and-country.mdx

```diff
@@ -141,13 +141,9 @@ as they would be downloaded from the account portal:
 
 ### MMDB example files
 
-We maintain test MMDB files on GitHub:
+TODO: Add Country
```

Comment:
A more flexible approach to `<MmdbFileExamples />` would be not to use props for the product + link , but to pass in array of objects.

```
<MmdbFileExamples files={[
    {
       filename: 'GeoIP2-City-Test.mmdb',
       link: 'https://github.com/maxmind/MaxMind-DB/blob/main/test-data/GeoIP2-City-Test.mmdb'
    }
  ]}
/>
```

Then in `<MmdbFileExamples />` you'd have something like
```
const fileItems = files.map((file) => <li><a href={file.link}>{file.filename}</a></li>
```

Reply:
CC: @christophermluna

Reply:
@kevcenteno, given that the product name, file names, and links are all straightforward manipulations of one another for both MMDB and CSV files, is it still preferable to pass an array of objects instead of an array of products?

for example:

- product name: Anonymous IP
- CSV file name: GeoIP2-Anonymous-IP-CSV_Example.zip
- CSV path: https://dev.maxmind.com/static/GeoIP2-Anonymous-IP-CSV_Example.zip
- MMDB file name: GeoIP2-Anonymous-IP-Test.mmdb
- MMDB path: https://github.com/maxmind/MaxMind-DB/blob/main/test-data/GeoIP2-Anonymous-IP-Test.mmdb

is it better to pass all these values distinctly, or pass the product name and manipulate that string as needed to get the variations?

Reply:
I would discourage having a catch-all component (`<FileExamples />`) that handles both CSV and MMDB examples. The text content and component API is different enough that we'd have a bunch of conditions that would be more complicated to use and develop vs two focused and discrete components (`<MmdbFileExamples />` and `<CsvFileExamples />`).

I would also discourage using string manipulation to extract/generate product or file names.  It's simple enough to add another object property like `productName`, and we'd avoid having mapping logic for products whose associated filename could be very different.

Reply:
thanks for the guidance, kevin!